src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -52,4 +52,45 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Content Remove="$(DefaultRemoves)" />
   </ItemGroup>
 
+  <UsingTask TaskName="CheckForDuplicateItems" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+
+  <Target Name="CheckForDuplicateItems" BeforeTargets="_CheckForInvalidConfigurationAndPlatform">
+
+    <PropertyGroup>
+      <!-- TODO: Create a fwlink for this -->
+      <DefaultItemsMoreInformationLink>https://github.com/dotnet/sdk</DefaultItemsMoreInformationLink>
+
+      <DefaultPackageReferencesEnabled Condition="'$(DisableImplicitFrameworkReferences)' != 'true'">true</DefaultPackageReferencesEnabled>
+    </PropertyGroup>
+
+    <CheckForDuplicateItems
+      Items="@(Compile)"
+      ItemName="Compile"
+      DefaultItemsEnabled="$(EnableDefaultItems)"
+      DefaultItemsOfThisTypeEnabled="$(EnableDefaultCompileItems)"
+      PropertyNameToDisableDefaultItems="EnableDefaultCompileItems"
+      MoreInformationLink="$(DefaultItemsMoreInformationLink)"
+      />
+
+    <CheckForDuplicateItems
+      Items="@(EmbeddedResource)"
+      ItemName="EmbeddedResource"
+      DefaultItemsEnabled="$(EnableDefaultItems)"
+      DefaultItemsOfThisTypeEnabled="$(EnableDefaultEmbeddedResourceItems)"
+      PropertyNameToDisableDefaultItems="EnableDefaultEmbeddedResourceItems"
+      MoreInformationLink="$(DefaultItemsMoreInformationLink)"
+      />
+    
+    <!-- TODO: Do we check for duplicate content items here, or in the Web SDK? EnableDefaultContentItems isn't defined or used at all by the .NET SDK -->
```

Comment:
I'd say to do it here because `Content` is a type known in `Common.targets`, and this is the closest layer above that. If it were defined and used only in Web, I'd say to move it there.